clang/include/clang/Basic/CodeGenOptions.h

```diff
@@ -78,6 +78,13 @@ class CodeGenOptions : public CodeGenOptionsBase {
     SRCK_InRegs    // Small structs in registers (-freg-struct-return).
   };
 
+  enum ComplexArgumentConventionKind {
+    CMPLX_Default,
+    CMPLX_OnStack,
```

Comment:
it looks you define the `CMPLX_Default` and   ` CMPLX_OnStack` , but never use them.

Reply:
This was for future reference, if any arch wants to use this.